Added a few Sanctum Cards

Achocholacek opened the pull request:
Added [Sanctum] Bordan the Redeemed
Added [Sanctum] Lion Bautrem
Added [Sanctum] The Grey Rider

(Tested fine locally. Hope I'm doing this right!)

cryogen commented:
Thanks!

Can you remove the comments after the card ids?

These are a legacy and should he removed going forwards

jeremylarner commented on server/game/cards/02-AoA/TheGreyRider.js:
```diff
@@ -0,0 +1,49 @@
+class TheGreyRider extends Card {
+    setupCardAbilities(ability) {
+        this.play({
+            optional: true,
```
you can just pass `fight: true, reap: true` to this.play so you don't need to do the same thing three times

Achocholacek commented:
Cleaned up comments as requested. Added new cards. Couple non working ones (Aubade and Abond - they both look like they SHOULD work, but no such luck.)

cryogen commented on server/game/cards/01-Core/SelwynTheFence.js:
```diff
@@ -5,7 +5,8 @@ class SelwynTheFence extends Card {
             reap: true,
             target: {
-                cardType: 'creature',
+                activePromptTitle: 'Choose a captured aember to move to your pool.',
```
If you use 'amber' it will automatically be converted to Aember and a amber icon by the client

cryogen commented on server/game/cards/02-AoA/AubadeTheGrim.js:
```diff
@@ -0,0 +1,24 @@
+        this.reap({
+            condition: context => context.source.hasToken('amber'),
+            effect: 'move 1 amber from Aubade the Grim to the pool',
+            then:{
```
This doesn't need to be a "then", just a straight game action
